src/test_l1_dsVideoDevice.c

```diff
@@ -943,35 +910,19 @@ void test_l1_dsVideoDevice_positive_dsGetVideoCodecInfo(void)
 
     // Step 03: Get video codec information with dsVIDEO_CODEC_MPEGHPART2
     result = dsGetVideoCodecInfo(handle, dsVIDEO_CODEC_MPEGHPART2, &codecInfo);
-    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(result, dsERR_NONE);
-    #else
-    UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
-    #endif
 
     // Step 04: Get video codec information with dsVIDEO_CODEC_MPEG4PART10
     result = dsGetVideoCodecInfo(handle, dsVIDEO_CODEC_MPEG4PART10, &codecInfo);
-    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(result, dsERR_NONE);
-    #else
-    UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
-    #endif
 
```

Comment:
removing the check as it is not part of enhanced error code.
The api's are not implemented for Llama hence returning dsERR_OPERATION_NOT_SUPPORTED